lib/libeconf.map

```diff
@@ -68,9 +66,6 @@ LIBECONF_0.4 {
 } LIBECONF_0.3;
 LIBECONF_0.5 {
   global:
-    econf_readFileWithCallback;
```

Comment:
Why have you removed these three entries ? econf_readFileWithCallback,... is in the API. Where it should be defined then ?

Reply:
They're already defined under `LIBECONF_0.4`. The clang warning I resolved was that they were defined twice. As far as I know it's better to remove the duplicate entry under `LIBECONF_0.5` since it's a superset of 0.4 anyway.

Reply:
ah, sorry I have overseen it. Thanks for the fix !